### PR TITLE
Import rynovate into the renovate-config-seek namespace 

### PR DESCRIPTION
**Pull request by AaronMoat**
- Delete all our files
- Copy all files from https://github.com/seek-oss/rynovate/commit/598ee7d7eb32a8a992efce0673858f1f2ff7079e
- In another commit, find and replace rynovate -> renovate-config-seek 

This requires workshopping, but keen for feedback and direction.

As a short summary of conversations, `rynovate` has evolved to be the defacto standard across SEEK and it would be good to promote that to the renovate-config-seek namespace. 

There is more work to do here before widespread usage, but getting this initial diff in will help conversations

### CODE REVIEW

File: default.json
Comment (72636c) [inline default.json:1]: 1. Do we intend to take over the `default` here? An alternative may be to reroute `github>seek-oss/rynovate` to a named preset like `github>seek-oss/renovate-config-seek:recommended`.
2. Existing `renovate-config-seek` consumers go via npm package, so while we don't _need_ to reserve `default`, there is an open question about where we migrate them to (`default`, `third-party-major`, a separate preset that retains prior `renovate-config-seek` behaviour?)

File: package.json [hunk 1]
```diff
@@ -1,85 +1,21 @@
 {
-  "name": "renovate-config-seek",
```
Comment (72636c) [inline package.json:2]: I assume this means that we are moving from a versioned npm model (repos must explicitly upgrade to the new package version) to a GitHub default branch model (repos automatically receive merged changes). I like this change but is it contentious to anyone else?

File: package.json [hunk 2]
```diff
@@ -1,85 +1,21 @@
...
-    "commitizen": "^2.10.1",
-    "commitlint-config-seek": "^1.0.0",
-    "cz-conventional-changelog": "^2.1.0",
-    "husky": "^0.14.3",
-    "renovate": "^19.38.7",
-    "semantic-release": "^15.5.5"
+  "scripts": {
+    "format": "prettier --write .",
+    "lint": "prettier --check .",
+    "test": "concurrently yarn:test:*",
```
Comment (72636c) [inline package.json]: Nitpick: should we pnpm?

```suggestion
    "test": "pnpm /^test:.+/",
```

File: README.md [hunk 3]
```diff
@@ -1,41 +1,174 @@
...
-- Commit messages and pull request titles are specially formatted to follow our [commitlint-config-seek](https://github.com/seek-oss/commitlint-config-seek) conventions.
+[![Validate](https://github.com/seek-oss/renovate-config-seek/actions/workflows/validate.yml/badge.svg)](https://github.com/seek-oss/renovate-config-seek/actions/workflows/validate.yml)
+
+Mildly-aggressive [Renovate] presets for keeping dependencies up to date.
+
+See [renovate-config-seek] for a baseline preset that only maintains SEEK npm packages.
+
+[renovate]: https://renovatebot.com/
+[renovate-config-seek]: https://github.com/seek-oss/renovate-config-seek
+
```
Comment (72636c) [inline README.md]:
```suggestion
[renovate]: https://renovatebot.com/

```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -0,0 +1 @@
+* @seek-oss/indirect
```
Comment (72636c) [inline .github/CODEOWNERS:1]: Reminder to change this

**Review by 72636c — APPROVED** [review]

File: default.json
Comment (AaronMoat) [reply default.json:1]: 1. No strong opinions personally
2. There are very few consumers and they'll need to actively migrate away. That doesn't answer your question. I know Jahred was keen on having a preset that has similar functionality to the existing, which I'm open to doing in this PR or a follow up

File: package.json (hunk 1 above)
Comment (AaronMoat) [reply package.json:2]: They're deprecated, so I hope not: https://docs.renovatebot.com/config-presets/#npm-hosted-presets

File: package.json (hunk 2 above)
Comment (AaronMoat) [reply package.json]: TIL

File: .github/CODEOWNERS (hunk 4 above)
Comment (AaronMoat) [reply .github/CODEOWNERS:1]: Open to changing this in the PR if we can come to an agreement

File: default.json
Comment (72636c) [reply default.json:1]: > I know Jahred was keen on having a preset that has similar functionality to the existing

I'm keen to interrogate this use case over an alternative like `third-party-major`. When is it acceptable for a system owner to ignore major revisions of their third-party dependencies from a tech health perspective?

Comment (AaronMoat) [reply default.json:1]: I hadn't really looked at third-party-major before but it seems like a fairly reasonable alternative to the old renovate-seek-config, to me (majors + seek package minor/patches)

File: .github/CODEOWNERS (hunk 4 above)
Comment (AaronMoat) [reply .github/CODEOWNERS:1]: What about:

```suggestion
* @seek-oss/sig-backend-tooling @seek-oss/sku-maintainers 
```

(and adjusting the admin access as per ^)

@jahredhope @72636c

**Pull request closed without merging**